data/shubhamku044.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "Shubham Kumar",
+  "bio": "Open Source | Full stack developer",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "ReactJS",
+    "TailwindCSS"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/shubhamku044"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/shubhamku044"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://www.youtube.com/@shubhamku044"
+    },
+    {
```

[Recommendation] Consider placing the icons after the URLs. It'll look more appealing.
```suggestion
      "url": "https://twitter.com/shubhamku044"
      "icon": "FaTwitter"
    },
    {
      "url": "https://github.com/shubhamku044"
      "icon": "FaGithub"
    },
    {
      "url": "https://www.youtube.com/@shubhamku044"
       "icon": "FaYoutube"
    },
    {
```

---

Thank you @CBID2 for reviewing, it is unclear what you mean by "It'll look more appealing"? People don't look at each other's json file (unless reviewing), they look at the rendered web page - please note this type of feedback on someone's PR is distracting and can be off-putting to the author, so I will mark as resolved